tssc/step_implementers/__init__.py

```diff
@@ -29,5 +30,6 @@
     'container_image_static_vulnerability_scan',
     'uat',
     'canary_test',
-    'deploy'
+    'deploy',
+    'validate_environment_configuration'
```

Comment:
These are out of order compared to https://rhtconsulting.github.io/tsc-docs/ and should probably be updated.

Reply:
The order in that file doesn’t matter. It’s just listing files to be sure things do get imported if ever using import *

Reply:
Correct. The order doesn't _technically_ matter. However, having some order can be helpful for future readers/developers. This can either be alphabetical or another taxonomy. In this case the original order was following the tsc-docs diagram, so I figure it only makes sense to maintain that order.

Reply:
Hunter, in the tsc-docs diagram validate environment configuration is the last step following deploy.   Are you saying the lines previous to the change are out of order?

Reply:
Changed the order to : Deploy, Validate, UAT, canary.